openfisca_web_api/application.py

```diff
@@ -74,7 +74,7 @@ def x_api_version_header_setter(app):
     def set_x_api_version_header(environ, start_response):
         req = webob.Request(environ)
         res = req.get_response(app)
-        res.headers.update({'X-API-Version': environment.git_head_sha})
```

Comment:
Maybe we should just remove this, as we don't use the header to give version information.